admin/test-brew-recipe

```diff
@@ -1,11 +1,9 @@
 #!/usr/bin/env python
-# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+# Copyright 2015 ClusterHQ.  See LICENSE file for details.
```

Comment:
:question: Probably don't have the year. Is it elsewhere / standard / necessary? Is there a method for updating it annually? Or an issue to create one?